networkx/__init__.py

```diff
@@ -51,7 +51,9 @@ def __getattr__(name):
     raise AttributeError(f"module {__name__} has no attribute {name}")
 
 
-# These are import orderwise
+# These are imported in order as listed
+from networkx import lazy_imports
```

Comment:
Why not use lazy imports to make this lazily accessible? :smile:

Maybe expose `nx.lazy_load` as a function?